src/test/java/com/adyen/util/CertificateUtilTest.java

```diff
@@ -50,4 +55,19 @@ public void testLoadCertificateFromInputStream() throws CertificateException {
 
         Assert.assertNotNull(certificate);
     }
+
```

Comment:
minor: more tests can be added for other negative scenarios like if resource.getPath() is null or keystore type is null etc.

Reply:
Added null check and more tests.